src/content/architecture/gateway-engine/_index.en.md

```diff
@@ -6,35 +6,54 @@ weight = 2
 The Gateway Engine component is deployed in each participating cluster and
 is responsible for establishing secure tunnels to other clusters.
 
+The Gateway Engine has a pluggable architecture for the cable engine component
+that maintains the tunnels. The following implementations are available:
+
+* an IPsec implementation using [strongSwan](https://www.strongswan.org/) (via the
+  [goStrongswanVici](https://github.com/bronze1man/goStrongswanVici) library);
+  this is currently the default;
+* an IPsec implementation using [Libreswan](https://libreswan.org/);
+* an implementation for [WireGuard](https://www.wireguard.com/).
+
 Instances of the Gateway Engine run on specifically designated nodes in a
-cluster of which there may be more than one for fault tolerance. There is
-only one active Gateway Engine instance at a time in a cluster. They
-perform a leader election process to determine the active instance and the
-others await in standby mode ready to take over should the active instance
-fail.
+cluster of which there may be more than one for fault tolerance. Submariner
+supports High Availability active/passive for the Gateway Engine component,
+which means that there is only one active Gateway Engine instance at a time
+in a cluster. They perform a leader election process to determine the active
+instance and the others await in standby mode ready to take over should the
+active instance fail.
+
+{{% notice info %}}
+The Gateway Engine is deployed as a DaemonSet that is configured to only run
+on nodes labelled with "submariner.io/gateway=true".
+{{% /notice %}}
 
 The active Gateway Engine communicates with the central Broker to advertise
 its `Endpoint` and `Cluster` resources to the other clusters connected to the
 Broker, also ensuring that it is the sole `Endpoint` for its cluster. The
-active Gateway Engine also establishes a watch on the Broker to learn about
-the active `Endpoint` and `Cluster` resources advertised by the other clusters.
-Once two clusters are aware of each other's `Endpoints`, they can establish a
-secure tunnel through which traffic can be routed.
+Route Agent Pods running on the cluster learn about the local `Endpoint`,
+and will create a VxLAN tunnel along with routing rules that point to the active
+Gateway node for cross-cluster communication. The active Gateway Engine
```

Comment:
Perhaps we don't need as much detail here re: Route Agent and link to that section? 

```suggestion
[Route Agent](../route-agent/) Pods running in the cluster learn about the local `Endpoint` and setup the necessary infrastructure to route cross-cluster traffic from all nodes to the active Gateway Engine node. The active Gateway Engine
```